lesson25/main.js

```diff
@@ -19,14 +32,78 @@ submitBtn.addEventListener(
   false
 );
 
+checkBox.addEventListener(
+  "click",
+  (e) => {
+    e.target.checked ? (flags.checkBox = true) : (flags.checkBox = false);
```

Comment:
I learned a lot about ternary operators, thank you very much.
【Question】
How do you choose whether to use A or B?

Reply:
Thank you for your comment.

I use ternary operators only simple and the condition are two. 
When the conditions are complex (such as when the nesting of If statements becomes deep), it becomes difficult to understand, so I don't use.

I'm also curious about this, and I've searched. [Free Code Camp](https://www.freecodecamp.org/news/ternary-operator-javascript-if-statement-tutorial/) said "it's not recommended to replace multiple if/else statements with multiple ternary operators because it makes the code harder to read in the future. ". So I think it's good to use it only simple conditions.
[This page](https://qiita.com/smicle/items/7d3b9881834dc0142fb7) might also find this helpful.(Qiita)

Reply:
Thank you for your explanation and referring me to your reference article.
According to [this page](https://qiita.com/smicle/items/7d3b9881834dc0142fb7), would you say that the following is more appropriate?
```
flags.checkBox = e.target.checked ? true : false;
```

Reply:
Thank you for your idea. Don't be sorry please :)
Yeah I think it's also the correct way, and it works! 

When I read this code first time, it's in my head like this.
```
( flags.checkBox = e.target.checked ) ? true : false;
``` 

Because I thought the same way when I read Qiita.
```
( result = (str === 'piyo') ) ? 'piyo！！' : 'not piyo…'
```
So I couldn't understand and thought it doesn’t make sense.

But I realized actually this means `flags.checkBox = ( e.target.checked ? true : false; )`.
I've totally misunderstood since I was reading Qiita. Now I understand and got new idea, thanks to you.
I've refactored this part :)
[843b360](https://github.com/yuka830/js-lessons/pull/30/commits/843b360e92c0015524b690091e605b2226d18655)

Reply:
Thank you very much.
I'm glad I had the opportunity to learn about ternary operators as well.